<launch>
    <node pkg="arx_x5_controller" type="X5Controller" name="arm" output="screen">
        <param name="arm_control_type" value="normal_v1"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/arm_cmd"/>
        <param name="pub_topic_ee_name_v1" value="/arm_status_ee"/>
        <param name="pub_topic_joint_name_v1" value="/arm_status_joint"/>
    </node>
    <!-- <node pkg="arx_x5_controller" type="trajectory_generator" name="trajectory_generator" output="screen">

    </node> -->
</launch>